docs/book/application-integration/using-custom-elements-with-laminas-servicemanager.md

```diff
@@ -1,4 +1,4 @@
-# Advanced use cases
+# Using Custom Elements with laminas-servicemanager
```

Comment:
Unfortunately the headline does not match the content of the page because some more topics are covered then custom elements. The topics of the page should be split, but until this, please revert the changes.

(And please remember: renaming a page leads to a new URL and therefore a redirect must be added.)

Reply:
I disagree partly, everything is this page is related to laminas-servicemanager. How about naming the page "Usage with laminas-servicemanager"?

Alternatively, what would you suggest on how to split the page? And how can I add a redirect?

Reply:
> I disagree partly, everything is this page is related to laminas-servicemanager.

I referred to "Using Custom Elements". The topics "Short names" and "Handling dependencies" are not part of this.

> Alternatively, what would you suggest on how to split the page?

I like your suggestion about custom elements because this is an important topic that needs to be highlighted more.

> And how can I add a redirect?

At the moment only with an _unbeautiful_ script tag. See: https://github.com/laminas/documentation-theme/issues/59

Reply:
"unbeautiful" is indeed a good word for that... 😆